krabcake/rs_hello/src/lib.rs

```diff
@@ -727,7 +754,7 @@ pub extern "C" fn rs_trace_store(
                 addr,
                 shadow_data,
             );
-            TRACKED_ADDRS.push((addr, Tag(shadow_data as u64)));
+            TRACKED_ADDRS.push((addr, Tag::Id(shadow_data as u64)));
```

Comment:
see nearby note about `Tag::Id(s1 as u64)` being a suspect pattern; probably needs replacement with the decoder helper function